src/resolve/resolved-dns-cache.c

```diff
@@ -533,6 +543,10 @@ static int dns_cache_put_negative(
         assert(key);
         assert(owner_address);
 
+        /* Don't store the negative responses if the 'ServeStale' flag is enabled. */
+        if(enable_serve_statle && rcode != DNS_RCODE_NXDOMAIN)
```

Comment:
follow coding style, i.e. a space before the `(`

Reply:
Fixed